src/Pokémon.java

```diff
@@ -0,0 +1,52 @@
+package src;
+
+public abstract class Pokémon {
```

Comment:
Oei! een accent aigu in een klasse naam zou ik niet doen. Ik vind het bijzonder dat Java dat toelaat. 
Technisch is het geen enkel probleem, blijkbaar, maar het is erg onconventioneel en ook niet elk toetsenbord kan het teken maken, waardoor sommige mensen de klasse niet zouden kunnen gebruiken.